### PR TITLE
chore(hacbs-1077): add boilerplate task & pipeline runs for test

### PR DESCRIPTION
**Pull request by parasense**
* Each pipeline & task version was given a boilerplate run.yaml
* These could also potentially serve as the sample runs
* The idea is tests will become more elaborate later

Signed-off-by: Jon Disnard <jdisnard@redhat.com>

### CODE REVIEW

**Comment by parasense** [comment]
This PR does not touch CI.
These tests are mostly symbolic, because (for now) the tests would have to be run manually.
Hopefully these serve as a good starting point, and germinate to more fulfilling & comprehensive set of tests in future.

File: catalog/pipeline/release/0.1/tests/run.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      value: ""
+    - name: extraConfigGitRevision
+      value: ""
+    - name: extraConfigPath
+      value: ""
+    - name: postCleanUp
+      value: ""
+  pipelineRef:
+    name: release
+    bundle: quay.io/release-service/pipeline-release:0.1@main
```
Comment (scoheb) [inline catalog/pipeline/release/0.1/tests/run.yaml:22]: how will this work in a CI run? will the bundles get overridden by newly built artifacts?

Comment (parasense) [reply catalog/pipeline/release/0.1/tests/run.yaml:22]: Using this pipelineRef, yes.
It should pull the image tagged "0.1" and "main", but of course one could use a digest, or the GitHub repo's commit sha1.
For the sake of simplicity, this definition uses the version tag + branch-ref tag

Comment (parasense) [reply catalog/pipeline/release/0.1/tests/run.yaml:22]: FYI - This pipeline was recently pushed.
Take a look here:
https://quay.io/repository/hacbs-release/pipeline-release?tab=tags

And, I see a mistake, so I'm going to push an updated commit. fyi `s/hacbs-release/release-service/`

**Comment by parasense** [comment]
FYI - [This was a rebase](https://github.com/hacbs-release/release-bundles/pull/50#event-7357449591) against origin/main, and re-signed the GPG signature for HEAD commit. Otherwise there would be merge-drama involving GPG signatures.

File: catalog/pipeline/release/0.1/tests/run.yaml (hunk 1 above)
Comment (parasense) [reply catalog/pipeline/release/0.1/tests/run.yaml]: To finalize this conversation. 
We will be removing these extra tags.
We will be using only the catalog version (e.g., 0.2) of the resource, the image digest, or both.
(resolving)

**Review by scoheb — APPROVED** [review]
Looks good. I assume a developer will have to be in charge of bumping the ref version in the pull spec. Seems acceptable

**Pull request merged**